app/models/participacao.rb

```diff
@@ -1,6 +1,7 @@
 class Participacao < ActiveRecord::Base
   belongs_to :usuario
   belongs_to :acao
+  delegate :nome, :to => :acao, :allow_nil => false
```

Comment:
http://www.virtuouscode.com/2011/07/05/demeter-its-not-just-a-good-idea-its-the-law/